app/viewmodels/checkAnswers/AssessmentsSummary.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package viewmodels.checkAnswers
+
+import controllers.routes
+import models.AssessmentAnswer.NoExemption
+import models.ott.CategoryAssessment
+import models.{CheckMode, UserAnswers}
+import pages.AssessmentPage
+import play.api.i18n.Messages
+import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
+import viewmodels.govuk.summarylist._
+import viewmodels.implicits._
+
+object AssessmentsSummary {
+
+  def row(
+    recordId: String,
+    answers: UserAnswers,
+    assessment: CategoryAssessment,
+    indexOfThisAssessment: Int,
+    numberOfAssessments: Int,
+  )(implicit messages: Messages): Option[SummaryListRow] = {
+    val b = answers.get(AssessmentPage(recordId, indexOfThisAssessment)).flatMap { answer =>
```

Comment:
call it something other than b!!!

Reply:
It's just a debug placeholder so I was meant to remove it anyway

Reply:
Removed